drydock/templates/kustomized/tutor13/base/k8s/drydock-jobs/minio.yml

```diff
@@ -0,0 +1,31 @@
+{%- if "minio" in DRYDOCK_INIT_JOBS %}
```

Comment:
This may be a bit hacky, but one way to know if a plugin like this is installed is to check `if MINIO_DOCKER_IMAGE is defined`. Then we add all the files but they will be empty if the plugin isn't enabled. What do you think?

Reply:
Thanks! :dancers: